app/controllers/confirmations_controller.rb

```diff
@@ -14,6 +14,7 @@ class ConfirmationsController < ApplicationController
   before_action :require_cookies, :only => [:confirm]
 
   def confirm
+    # The post happens in confirm.html.erb and user.js
```

Comment:
I don't think this is very useful - even if you accept it's true right now, and that depends on how you define "happens" exactly, there's no guarantee that those locations will stay the same and nobody will find and update this if they change one of those files.